docs/md/docs/2020-11-02-turn-the-senior-around.md

```diff
@@ -0,0 +1,68 @@
+# Turn The Senior Around
+
+As engineering managers we take many considerations into account, two of the main ones are:
+- Delivering on time
+- Make our people grow
+
+During my experience as an engineering manager and as an engineer all the managers I have met saw those subjects critical and invested a lot in improving them.<br>
```

Comment:
How does this sentence sound?
`These two responsibilities were considered critical amongst all the engineering managers I have met with throughout my career as an Engineering Manager and as a Software Engineer.`